### PR TITLE
Add Python 3.6

### PR DESCRIPTION
**Pull request by hugovk**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,5 +33,8 @@
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python:: Implementation:: CPython',
```
Comment (ThiefMaster) [inline setup.py]: missing space before `::` (also in the next line)

Comment (hugovk) [reply setup.py]: Thanks, reset, fixed, and force pushed.

**Comment by hugovk** [comment]
The Python 2.6 and 3.3 failures are unrelated to this PR, see https://github.com/pallets/meta/issues/24.

Here's the pip installs for itsdangerous from PyPI for the last month (via `pypinfo --percent --pip itsdangerous pyversion`) -- more than 50% for Python 3! 👍 

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |   47.1% |        746,612 |
| 3.6            |   25.2% |        398,831 |
| 3.5            |   21.0% |        333,045 |
| 3.4            |    5.9% |         94,266 |
| 2.6            |    0.5% |          8,401 |
| 3.3            |    0.1% |          2,115 |
| 3.7            |    0.1% |          1,802 |
| 3.2            |    0.0% |            133 |
| None           |    0.0% |             15 |

**Comment by lepture** [comment]
@davidism you can close this PR now.

**Pull request closed without merging**